packages/docs/README.md

```diff
@@ -97,6 +97,7 @@ Browse the code, raise an issue, or contribute a PR:
 
 Try Celo out:
 
+- [Using the Web Wallet](getting-started/using-the-web-wallet.md) on the Alfajores Testnet
```

Comment:
this is not a main net production ready web wallet. so I don't think it should be in the main README page.